winconf-mingw.pri

```diff
@@ -17,11 +17,11 @@ CONFIG(debug, debug|release) {
 
 RC_FILE = qbittorrent_mingw.rc
 
-#You need to link with libtorrent > 0.15.5 (or svn) and you must
-#configure libtorrent to use iconv in the building process. This is
-#needed for correct Unicode support.
+# You need to link with libtorrent > 0.15.5 (or svn) and you must
```

Comment:
@Chocobo1 since we're doing maintenance can you remove this comment block altogether?